geoips/commandline/geoips_describe.py

```diff
@@ -19,17 +19,17 @@
 from geoips import interfaces
 
 
-class GeoipsGetInterface(GeoipsExecutableCommand):
-    """Get Command which retrieves information about a from or about a GeoIPS interface.
+class GeoipsDescribeArtifact(GeoipsExecutableCommand):
+    """Command which describes information about a GeoIPS artifact.
```

Comment:
Maybe "Command which returns information describing a GeoIPS artifact."?

Reply:
Sure, I'll make that change. I thought that sounded weird too.

Reply:
Fixed.